src/plugins/trace/dll/DataModel/ETW/QuicEtwEvent.cs

```diff
@@ -115,6 +115,12 @@ private static QuicObjectType ComputeObjectType(TraceEvent evt)
                 case QuicEventId.LibraryInitialized:
                     if (QuicEvent.ParseMode != QuicEventParseMode.Full) return null;
                     return new QuicLibraryInitializedEvent(timestamp, processor, processId, threadId, pointerSize, data.ReadUInt(), data.ReadUInt());
+                case QuicEventId.LibraryInitializedV2:
```

Comment:
Missing the new rundown events in these files.

Reply:
yeah, noted in my own comment.